### ISSUE TITLE
T5 Preview: Accessibility Checker doesn't work if toolbar is set to false

### ISSUE THREAD

**Issue by androb** [tinymce/tinymce#4620]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
when toolbar is set to false (and the menu is on), the accessibility checker does not work. It just highlights the table as having an accessibility issue but shows no UI.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tinymce.com](http://fiddle.tinymce.com/) or similar.**

**What is the expected behavior?**
The Accessibility Checker UI is shown.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
T5 Preview 2 build 2

**Comment by TheSpyder** [comment]
Is this a regression from Tiny 4?

**Comment by TheSpyder** [comment]
Moved to our internal tracker as this relates to a closed source plugin.
